contracts/mech_usage/MechActivityChecker.sol

```diff
@@ -10,42 +10,42 @@ interface IMultisig {
     function nonce() external view returns (uint256);
 }
 
-// AgentMech interface
-interface IAgentMech {
-    /// @dev Gets the requests count for a specific account.
-    /// @param account Account address.
-    /// @return requestsCount Requests count.
-    function getRequestsCount(address account) external view returns (uint256 requestsCount);
+// Mech Marketplace interface
+interface IMechMarketplace {
+    /// @dev Gets deliveries count for a specific operator.
+    /// @param operator Operator address.
+    /// @return Deliveries count.
+    function getOperatorDeliveriesCount(address operator) external view returns (uint256);
 }
 
-/// @dev Provided zero mech agent address.
-error ZeroMechAgentAddress();
+/// @dev Provided zero address.
+error ZeroAddress();
 
-/// @title MechAgentMod - Smart contract for AI agent mech staking modification
+/// @title MechActivityChecker - Smart contract for mech staking activity checking
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
 /// @author Andrey Lebedev - <andrey.lebedev@valory.xyz>
 /// @author Mariapia Moscatiello - <mariapia.moscatiello@valory.xyz>
 contract MechActivityChecker is StakingActivityChecker{
-    // AI agent mech contract address.
-    address public immutable agentMech;
+    // AI agent mech marketplace contract address.
+    address public immutable mechMarketplace;
 
-    /// @dev MechAgentMod constructor.
-    /// @param _agentMech AI agent mech contract address.
+    /// @dev MechActivityChecker constructor.
+    /// @param _mechMarketplace AI agent mech marketplace contract address.
     /// @param _livenessRatio Liveness ratio in the format of 1e18.
-    constructor(address _agentMech, uint256 _livenessRatio) StakingActivityChecker(_livenessRatio) {
-        if (_agentMech == address(0)) {
-            revert ZeroMechAgentAddress();
+    constructor(address _mechMarketplace, uint256 _livenessRatio) StakingActivityChecker(_livenessRatio) {
+        if (_mechMarketplace == address(0)) {
+            revert ZeroAddress();
         }
-        agentMech = _agentMech;
+        mechMarketplace = _mechMarketplace;
     }
-
+    
     /// @dev Gets service multisig nonces.
     /// @param multisig Service multisig address.
-    /// @return nonces Set of a nonce and a requests count for the multisig.
+    /// @return nonces Set of a nonce and a deliveries count for the multisig.
     function getMultisigNonces(address multisig) external view virtual override returns (uint256[] memory nonces) {
         nonces = new uint256[](2);
         nonces[0] = IMultisig(multisig).nonce();
-        nonces[1] = IAgentMech(agentMech).getRequestsCount(multisig);
+        nonces[1] = IMechMarketplace(mechMarketplace).getOperatorDeliveriesCount(multisig);
```

Comment:
We care about the mech operator deliveries, not about the mech itself (that could be more than one or swapped).